Customer.AppServices/Validator/UpdateCustomerDtoValidator.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Customer.AppModels.Dtos;
+using FluentValidation;
+
+namespace Customer.AppServices.Validator
+{
+    public class UpdateCustomerDtoValidator : AbstractValidator<UpdateCustomerDto>
+    {
+        public UpdateCustomerDtoValidator()
+        {
+            RuleFor(x => x.FullName).NotEmpty().WithMessage("Currency required");
```

Comment:
1. O que é Currency?
2. Métodos próprios do FluentValidation já provem excelentes mensagens, não precisamos sobrescreve-las 
```suggestion
            RuleFor(x => x.FullName).NotEmpty();
```